content/articles/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/index.md

```diff
@@ -0,0 +1,425 @@
+
+In this tutorial, we will create a .NET desktop tool that shall enable us to enable or disable USB ports on our computers and also change the Read/Write permissions of the removable drives. We shall start by going through a detailed guide of creating the UI then we shall work on the code behind the above operations.
+
+### Prerequisites
+To be able to follow through this tutorial, you will need:
+1. [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the .NET framework.
+2. Basic knowledge in C# language and the .NET framework.
+3. A flash disk or any other removable drives for testing purposes.
+
+### Step 1 - Creating the project
+We shall start by opening Visual Studio and selecting `create new project` from the `Get started` options.
+ On the next window(Create a new project) : 
+
+![creating a new winforms project](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/create-a-new-project.jpg)
+
+ we will select the `Windows Forms App` template then select `Next`:  
+In the next window(Configure your new project) :
+
+![configure your new project](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/configure.jpg)
+
+We shall rename our project to `IOManager`, leave the other fields with their default values and select `Next`.
+In the `Additional information` window:
+
+![Additional information](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/additional_info.jpg)
+
+ we shall leave it to the default value and hit `Create`.
+After loading the project, you should see something similar to this:
+
+![initial window](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/initial_window.jpg)
+
+### Step 2 - Designing the UI
+
+#### 2.1 -Renaming the form title
+To rename the form title, click on  `Form 1` and in the `properties`  sidebar under `Text`, change to `I/O Manager`:
+
+![renaming Form1](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/renaming_Form1.jpg)
+
+#### 2.2 - Changing the background color
+Click the `I/O Manager` form and in the `Properties`  sidebar under `BackColor` change the background color. In this case, we shall change ours to `Teal`
+
+![changing form1 background color](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/form1_bgcolor.jpg)
+
+#### 2.3 - Changing form border
+This changes how the border of our form shall look: ie removes the `maximize` and `minimize` buttons. To do it, click the `I/O Manager` form and in the `Properties`  sidebar under `FormBorderStyle`, select `FixedToolWindow`:
+
+![form border style](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/form_border_style.jpg)
+
+#### 2.4 - Add Instructions label
```

Comment:
Only the first letter should begin with a capital. Fix where applicable throughout your article!